NitroxClient/GameLogic/Building.cs

```diff
@@ -1,213 +1,1326 @@
+                                }
+                            }
+                            else
+                            {
+                                // Case: the constructableBase doesn't use a ghostModel to be build, instead using its final objectModel to be build
+
+#if TRACE && BUILDING
+                                NitroxModel.Logger.Log.Debug("Constructable_NotifyConstructedChanged_Post - creating constructablebase without a ghost");
+#endif
+
+                                // Trying to find a Base in the parents of the gameobject itself
+                                Base aBase = instance.gameObject.GetComponentInParent<Base>();
+                                if (aBase != null)
+                                {
+#if TRACE && BUILDING
+                                    NitroxModel.Logger.Log.Debug("Constructable_NotifyConstructedChanged_Post - constructableBase has base in parentComponents: " + aBase);
+#endif
+                                    parentBaseId = NitroxEntity.GetIdNullable(aBase.gameObject);
+                                    if (parentBaseId != null)
+                                    {
+#if TRACE && BUILDING
+                                        NitroxModel.Logger.Log.Debug("Constructable_NotifyConstructedChanged_Post - constructableBase parentComponents base id: " + parentBaseId);
+#endif
+                                    }
+                                    else
+                                    {
+                                        parentBaseId = NitroxEntity.GetId(aBase.gameObject);
+#if TRACE && BUILDING
+                                        NitroxModel.Logger.Log.Debug("Constructable_NotifyConstructedChanged_Post - constructableBase parentComponentsbase had no id, assigned new one: " + parentBaseId);
+#endif
+                                    }
+                                }
+                            }
+
+                            Vector3 placedPosition = instance.gameObject.transform.position;
+
+                            NitroxId id = NitroxEntity.GetIdNullable(instance.gameObject);
+                            if (id == null)
+                            {
+
+                                id = NitroxEntity.GetId(instance.gameObject);
+#if TRACE && BUILDING
+                                NitroxModel.Logger.Log.Debug("Constructable_NotifyConstructedChanged_Post - constructableBase gameobject had no id, assigned new one: " + id);
+#endif
+
+                            }
+
+                            Transform camera = Camera.main.transform;
+                            Optional<RotationMetadata> rotationMetadata = rotationMetadataFactory.From(ghost);
+
+#if TRACE && BUILDING
+                            NitroxModel.Logger.Log.Debug("Constructable_NotifyConstructedChanged_Post - techType: " + instance.techType + " techType.Model(): " + instance.techType.Model());
+#endif
+
+                            //fix for wrong techType
```

Comment:
wrong tech type?

Reply:
I don't really know if this is a bug of subnautica or just intented to be so. If you want to build a Connector via Multiplayer-Builder you need the Enum Id = 5519. But when grabbing the TechType from a lokal builded Connector this has the Enum Id = 5508. Using this Id with the Multiplayer-Builder fails. So for saving or transfering to remote player changing tech type is needed. 
This is the only object i found, where the tech type changes happen.